hack/ci-e2e-kind-upgrade.sh

```diff
@@ -204,6 +204,13 @@ set_seed_name
 $(dirname "${0}")/download_gardener_source_code.sh --gardener-version $GARDENER_PREVIOUS_RELEASE --download-path $GARDENER_RELEASE_DOWNLOAD_PATH/gardener-releases
 export GARDENER_PREVIOUS_VERSION="$(cat $GARDENER_RELEASE_DOWNLOAD_PATH/gardener-releases/$GARDENER_PREVIOUS_RELEASE/VERSION)"
 
+# temporary fix of addresses in previous release (can be removed > v1.102)
```

Comment:
```suggestion
# TODO(MartinWeindel): Temporary fix of addresses in previous release for e2e upgrade tests (remove this after v1.102 has been released).
```

Otherwise, we might not find it 😉 ref https://github.com/gardener/gardener/blob/master/docs/development/process.md#todo-statements